dnsdist/conf/dnsdist.conf

```diff
@@ -23,8 +23,8 @@ addAction("internal.", RCodeAction(DNSRCode.REFUSED))
 --  for why a password is used instead of an API key.
 webserver("10.16.4.10:8083")
 setWebserverConfig({
-    password="we+ensure+security+via+network+segmentation",
-    apiKey="",
+    password="$scrypt$ln=10,p=1,r=8$pKoROW2iHTUujxtGGX7y/Q==$h3+OfsjwnslAcGMcHkO48j+5Ss7nry+WraiXEGIB1SM=",
+    apiKey="$scrypt$ln=10,p=1,r=8$pKoROW2iHTUujxtGGX7y/Q==$h3+OfsjwnslAcGMcHkO48j+5Ss7nry+WraiXEGIB1SM=",
```

Comment:
needs to be what's in https://github.com/desec-io/desec-ns/blob/99f6cadf4f1a9824420ca9f792b0770ab8f91c21/prometheus/conf/prometheus.yml#L23

Reply:
this is the hash of a random password. I reverted to `hashPassword()` for `password=` and kept the hash for `apiKey`.